static/panes/compiler.js

```diff
@@ -279,6 +285,27 @@ Compiler.prototype.initPanerButtons = function () {
         insertPoint.addChild(createOptView);
     }, this));
 
+    this.flagsButton = $(document.createElement('button'));
+    this.flagsButton.addClass('dropdown-item btn btn-light btn-sm');
+    this.flagsButton.html(
```

Comment:
Is this DOM manipulation necessary? Could we create the button directly in the corresponding section in `views/templates.pug` and then select it?

Reply:
We could; yes, but I uh... couldn't really understand what was going on in templates.pug or how to add it to that. I can give it another try though.

Reply:
If you get stuck, we can help: or else take on doing the small change to the PR to make it use pug separately.

Reply:
Any word on this? This is a nice improvement; can we help with the pug stuff? If it's not something you have time for, we can take it on completely - just let us know!

Reply:
I'm finally back from vacation, I hope to work on this this week or next.

Reply:
Okay; this should be done.